recipes/libsamplerate/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version = "0.1.8" %}
+
+package:
+  name: libsamplerate
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/erikd/libsamplerate.git
+  git_tag: 548c6761a0348a83c5f6ee68aa1581c855092195
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - autoconf
+    - automake
+    - libtool
+    - pkg-config
+
+test:
+  commands:
+    - true  # we perform testing in the build phase
```

Comment:
Can you test for a command or the existence of a library/file? I am not OK with this.